neuralmonkey/encoders/sentence_cnn_encoder.py

```diff
@@ -0,0 +1,288 @@
+from typing import Optional, Tuple, List, Any
+
+import tensorflow as tf
+from typeguard import check_argument_types
+
+from neuralmonkey.encoders.attentive import Attentive
+from neuralmonkey.model.model_part import ModelPart, FeedDict
+from neuralmonkey.logging import log
+from neuralmonkey.nn.noisy_gru_cell import NoisyGRUCell
+from neuralmonkey.nn.ortho_gru_cell import OrthoGRUCell
+from neuralmonkey.nn.highway import highway
+from neuralmonkey.dataset import Dataset
+from neuralmonkey.vocabulary import Vocabulary
+
+# pylint: disable=invalid-name
+RNNCellTuple = Tuple[tf.contrib.rnn.RNNCell, tf.contrib.rnn.RNNCell]
```

Comment:
Přečti si poznámky, co jsem napsal u enkodéru k pull requestu od @cifkao ( #363 ) - řekl bych, že to tu platí všechno taky.. :-)
@jlibovicky musíme s tim něco udělat, hodně se nám tu množní duplicitní kód. I duplicitní reviews. :-)